pkg/relay/relay.go

```diff
@@ -459,7 +459,8 @@ func (rs *Relay) prepareContents(submitBlockRequest *types.BuilderSubmitBlockReq
 				BlockNumber: s.Payload.Payload.Data.BlockNumber,
 				NumTx:       uint64(len(s.Payload.Payload.Data.Transactions)),
 			},
-			Timestamp: s.Payload.Payload.Data.Timestamp,
+			Timestamp:   uint64(time.Now().UnixMilli() / 1_000),
+			TimestampMs: uint64(time.Now().UnixMilli()),
```

Comment:
Recommend using `time.Duration` (which is what `UnixMilli()` returns, I think), rather than casting as `uint64`.

Reply:
`UnixMilli()` returns `int64`. How are you suggesting exactly to use `time.Duration`?

Reply:
Ah, my bad.  And on second thought, this isn't technically a duration.  Please disregard this comment.  Resolving.